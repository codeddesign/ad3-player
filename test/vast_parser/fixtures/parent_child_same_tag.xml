<?xml version="1.0" encoding="UTF-8"?>
<VAST xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.0">
  <VAST xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.0">
  </VAST>
  <ad id="1">
        <child>one</child>
        <child>two</child>
    </ad>
</VAST>
