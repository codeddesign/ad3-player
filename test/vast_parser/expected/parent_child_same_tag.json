{
  "vast": {
    "name": "VAST",
    "value": [
      {
        "vast": {
          "name": "VAST",
          "value": "",
          "attributes": {
            "xmlns:xsi": "http://www.w3.org/2001/XMLSchema-instance",
            "version": "1.0"
          }
        }
      },
      {
        "ad": {
          "name": "ad",
          "value": [
            {
              "child": {
                "name": "child",
                "value": "one",
                "attributes": {}
              }
            },
            {
              "child": {
                "name": "child",
                "value": "two",
                "attributes": {}
              }
            }
          ],
          "attributes": {
            "id": "1"
          }
        }
      }
    ],
    "attributes": {
      "xmlns:xsi": "http://www.w3.org/2001/XMLSchema-instance",
      "version": "2.0"
    }
  }
}
